README.md

```diff
@@ -174,20 +189,20 @@ builder.Services.AddOpenTelemetry()
 
 ### Manual Instrumentation usage
 
-In environments where no `IServiceCollection` is naturally available you may manually start instrumenting by creating an instance of `ElasticOpenTelemetryBuilder`. 
+In environments where an `IServiceCollection` is unavailable you may manually start instrumenting by creating 
```

Comment:
Just curious what's introducing this line breaks? It might be our line length in `.editorconfig`?

Reply:
Just me manually tweaking them. Istvan, in the past, told me there was a preferred width, and I've got into the habit of roughly sticking to that. It happens to be about the width I have when a preview is open side-by-side.

Reply:
Cool we should add this explicitly to `.editorconfig` at some point.